tests/test_smartJSON.py

```diff
@@ -111,3 +111,17 @@ def test_get_attrs_by_parent_w_simple_string(nested_json_one):
                     {'location': '5 Park Av, NY, NY 10010',
                         'email': 'alice@efgcompany.com',
                         'phone': '7564525797'}]}
+
+def test_empty_string():
+    empty_string = dict(smartJSON('""'))
+    assert  isinstance(empty_string, dict)
+
+def test_empty_object():
+    empty_object = dict(smartJSON('{}'))
+    assert isinstance(empty_object, dict) and not len(empty_object)
+
+def test_empty_string_as_value():
+    assert dict(smartJSON('{"a": ""}'))['a'] == ''
+
+def test_empty_object_as_value():
+    assert isinstance(dict(smartJSON('{"a": {}}'))['a'], dict)
```

Comment:
Like I said, the converted `dict` should contain `Node` instances, not plain strings. Please let me know if this makes sense or you would like me to clarify further.  Thanks again for taking the time to contribute! :)